Sources/PackageLoading/PkgConfig.swift

```diff
@@ -382,18 +386,24 @@ internal struct PCFileFinder {
     ///
     /// This is needed because on Linux machines, the search paths can be different
     /// from the standard locations that we are currently searching.
-    public init(brewPrefix: AbsolutePath? = .none) {
+    public init(brewPrefix: AbsolutePath? = .none, pkgConfig: AbsolutePath? = .none) {
```

Comment:
is this initializer only used in tests?

Reply:
Yes, it is only used in tests; I believe that `init(brewPrefix:)` was initially for tests only.  If not, that might be something that warrants re-visiting.

Reply:
if so lets make it internal rather than public? and maybe add a comment to say its for testing?

Reply:
Wait, maybe I'm misunderstanding something.  The type itself is already marked `internal`.  Does not make this an internal interface?  That was why I did this change this way - otherwise this is a problem as it is an ABI break and we should provide a function overload rather than a default valued parameter introduction.

Reply:
Wait, maybe I'm misunderstanding something. The type itself is already marked `internal`. Does not make this an internal interface? That was why I did this change this way - otherwise this is a problem as it is an ABI break and we should provide a function overload rather than a default valued parameter introduction.

Reply:
yes +1